Unreal/CarlaUE4/Plugins/Carla/Source/Carla/Sensor/Sensor.h

```diff
@@ -63,6 +63,11 @@ class CARLA_API ASensor : public AActor
     return Stream.IsStreamReady();
   }
 
+  bool AreClientsListening()
```

Comment:
The implementation should be moved tho the .cpp file.

Implementations on .hpp files increase the compilation time and are forcely inlined by the compiler.